app/views/users/settings/_shipping.html.haml

```diff
@@ -0,0 +1,71 @@
+.col-md-5.mb-40
+  .visible-xs.visible-sm
+    %h3.general-subtitle.mt-0= t('settings.shipping')
+  = form_for(@shipping, method: :post, remote: true, url: update_shipping_path) do |f|
+    .form-group
```

Comment:
seems that this form has the same fields that billing address. Can we use some partial for equal fields?